<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WRDRegItem</class>
 <widget class="QWidget" name="WRDRegItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="txt_Addr">
   <property name="geometry">
    <rect>
     <x>69</x>
     <y>14</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Input">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>391</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DataSelect">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>9</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据选择</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Quote">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>14</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>引用</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_SerialNum">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>14</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>序号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_Input">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>14</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
